composer.json

```diff
@@ -31,6 +31,7 @@
         "rector/rector-symfony": "^0.11.16",
         "rector/rector-phpoffice": "^0.11.2",
         "sebastian/diff": "^4.0.4",
+        "spatie/enum": "^3.9",
```

Comment:
This dependency should be in require-dev, as Rector itself is not using it.

Reply:
Thanks @TomasVotruba - I was wondering if that was the case.
Was trying to wrap my head around if it's actually required at all.